src/main/java/com/scylladb/alternator/StreamsAdapterDemoHelper.java

```diff
@@ -83,7 +86,18 @@ public static DescribeTableResult describeTable(AmazonDynamoDB client, String ta
     }
 
     public static ScanResult scanTable(AmazonDynamoDB dynamoDBClient, String tableName) {
-        return dynamoDBClient.scan(new ScanRequest().withTableName(tableName));
+        ScanResult r = dynamoDBClient.scan(new ScanRequest().withTableName(tableName));
```

Comment:
I wish we could have a better counter, then doing a full scan
ItemCount, which we don't have might help, even dynamodb only promised it would be updated once in 6h
https://github.com/scylladb/scylla/issues/5026